### PR TITLE
improve progress bar

### PR DESCRIPTION
**Pull request by xibr**
for more info see https://github.com/yausername/youtubedl-android/issues/147 and https://github.com/yausername/youtubedl-android/pull/148

before

https://user-images.githubusercontent.com/20098154/168210349-227d1999-3d18-4fac-bc4a-1d7801964bca.mp4

after

https://user-images.githubusercontent.com/20098154/168210786-43127631-2197-40e7-b2dc-1b508a423832.mp4

### CODE REVIEW

**Comment by yausername** [comment]
Video looks good. I will review the code later today.

**Comment by xibr** [comment]
Well, I think this is ready.

File: app/src/main/java/org/yausername/dvd/work/CommandWorker.kt [hunk 1]
```diff
@@ -74,7 +74,7 @@ class CommandWorker(appContext: Context, params: WorkerParameters) :
                 NotificationCompat.BigTextStyle()
                     .bigText(line)
             )
-            .setProgress(100, progress, false)
+            .setProgress(100, progress, progress == 0)
```
Comment (yausername) [inline app/src/main/java/org/yausername/dvd/work/CommandWorker.kt:77]: should this be `progress == -1.0f`

File: app/src/main/java/org/yausername/dvd/work/DownloadWorker.kt [hunk 2]
```diff
@@ -123,9 +124,9 @@ class DownloadWorker(appContext: Context, params: WorkerParameters) :
             .setContentTitle(name)
             .setStyle(
                 NotificationCompat.BigTextStyle()
-                    .bigText(line)
+                    .bigText(text)
             )
-            .setProgress(100, progress, false)
+            .setProgress(100, progress, progress == 0)
```
Comment (yausername) [inline app/src/main/java/org/yausername/dvd/work/DownloadWorker.kt:129]: same

File: app/src/main/java/org/yausername/dvd/work/CommandWorker.kt (hunk 1 above)
Comment (xibr) [reply app/src/main/java/org/yausername/dvd/work/CommandWorker.kt:77]: Should be `progress == -1` and not `progress == -1.0f` or am i wrong?

Comment (yausername) [reply app/src/main/java/org/yausername/dvd/work/CommandWorker.kt:77]: Yeah, you are right.

**Pull request merged**